### DISCUSSION TITLE
Does bun cli have an equivalent to the --require flag in Node?

### DISCUSSION THREAD

**Discussion by pmrotule** [oven-sh/bun#5016] [Q&A] [answered]
The [`--require`](https://nodejs.org/docs/latest/api/cli.html#-r---require-module) flag (or `--import` for ESM) in Node.js cli is pretty useful to preload a file before running the main entry file. Running one after the other wouldn't have the same effect as they would be running in different processes which wouldn't work for running the [NewRelic agent](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent/) for instance.

```
node -r newrelic ./dist/server.js
```

I was wondering if Bun has something equivalent. Should I rather create a feature request?

**Comment by pmrotule** [comment, marked as answer]
Sorry for the noise, I found it by running `bun --help`. I realize now that there is a lot of flags that are not in the documentation, could be nice to add a comment like "run `bun --help` for the complete list of options".

<img width="841" alt="Screenshot 2023-09-11 at 20 24 58" src="https://github.com/oven-sh/bun/assets/10983258/9c3d46e4-8916-48e3-930d-c2d4355be079">

**Reply by RichardBray** [reply]
It is mentioned in the documentation, just under the [bunfig.toml](https://bun.sh/docs/runtime/bunfig#preload) section 🤦

**Reply by i3Cheese** [reply]
It is not obvious that you can pass these options as arguments. I thought they were bunfig.toml exclusive :)
